runtime/v2/logging/logging_windows.go

```diff
@@ -0,0 +1,98 @@
+// +build windows
+
+/*
+   Copyright The containerd Authors.
+
+   Licensed under the Apache License, Version 2.0 (the "License");
+   you may not use this file except in compliance with the License.
+   You may obtain a copy of the License at
+
+       http://www.apache.org/licenses/LICENSE-2.0
+
+   Unless required by applicable law or agreed to in writing, software
+   distributed under the License is distributed on an "AS IS" BASIS,
+   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+   See the License for the specific language governing permissions and
+   limitations under the License.
+*/
+
+package logging
+
+import (
+	"context"
+	"fmt"
+	"net"
+	"os"
+	"os/signal"
+	"syscall"
+
+	"github.com/Microsoft/go-winio"
+	"github.com/pkg/errors"
+)
+
+// Run the logging driver
+func Run(fn LoggerFunc) {
+	err := runInternal(fn)
+	if err != nil {
+		fmt.Fprintln(os.Stderr, err)
+		os.Exit(1)
+	}
```

Comment:
I think the only thing missing is we should explicitly `os.Exit(0)` in the case where no error is returned. That will match the unix behavior where `Run` never returns to its caller.